pallocator/Cargo.toml

```diff
@@ -5,5 +5,13 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
-[dependencies]
-pgx = { version = "=0.5.0-beta.0", features = ["pg14"] }
+[dependencies.pgx-pg-sys]
+# pallocator only expects to be included in a graph which already features a pgx crate with:
+# - a crate version
+# - a "{PG_VERSION}" feature (which will probably be pg14)
+# Thus, we only need to make sure we enable the postgrestd feature for pgx-pg-sys,
+# and ask only for a version of pgx-pg-sys where that is possible.
+version = ">=0.5.0-beta.0"
+features = ["postgrestd"]
```

Comment:
Will the lack of a `pg14` here possibly resulot in some complicates if `resolver = "2"` is ever used?

Reply:
My understanding is that `resolver = "2"` is already the default if `edition = "2021"` is set, and it seems to have no negative effect in my testing so far. We can revisit this if it does.